frontend/src/components/navbar/top-navbar.tsx

```diff
@@ -11,83 +10,194 @@ interface TopNavBarProps {
+        width: '100%',
+        height: '3px',
+        transition: 'background-color 0.3s ease',
     },
-})
+    '&.active::after': {
+        backgroundColor: colors.green,
+    },
+    '&:hover::after': {
+        backgroundColor: colors.green,
+    },
+});
 
-const AdminLinks = loadAsync('Admin Links', () => import('./admin-links'))
-const AccountLinks = loadAsync('Account Links', () => import('./account-links'))
 
-export const TopNavBar: FCWOC<TopNavBarProps> = ({ children, className }) => {
-    const user = useCurrentUser()
-    const hideBanner = user.isResearcher
+const AdminLinks = loadAsync('Admin Links', () => import('./admin-links'));
+const AccountLinks = loadAsync(
+    'Account Links',
+    () => import('./account-links')
+);
+
+export const TopNavBar: React.FC<TopNavBarProps> = ({ className }) => {
+    const user = useCurrentUser();
+    const hideBanner = user.isResearcher;
+    const isMobile = useIsMobileDevice();
 
     return (
-        <nav className={cx('navbar', 'navbar-light', className)}>
-            <div className="navbar-dark bg-dark py-1">
-                <Container>
-                    <Box justify="between" align="center" gap padding={{ vertical: 'default' }}>
-                        <NavbarLogoLink />
-                        {children}
-                        <Box gap="xlarge" align="center">
-                            <Link to="/studies" css={{ color: 'white', textDecoration: 'none', fontWeight: 'bold' }}>
-                                Dashboard
-                            </Link>
-                            <DesktopResearcherLinks />
+        <nav
+            className={cx('navbar', 'navbar-light', className)}
+            style={{
+                backgroundColor: colors.navy,
+                color: 'white',
+                height: '80px',
+            }}
+        >
+            <div style={{ height: '80px', backgroundColor: colors.navy }}>
+                <Container style={{ height: '80px' }}>
+                    <Group
+                        justify='space-between'
+                        align='center'
+                        style={{
+                            width: '100%',
+                            height: '80px',
+                        }}
+                    >
+                        <Group>
+                            <NavbarLogoLink />
+                        </Group>
+                        <Group
+                            justify='center'
+                            align='center'
+                            gap='40px'
+                            style={{
+                                height: '80px',
+                                padding: '26px 0px 26px 0px',
+                            }}
+                        >
+                            {!isMobile &&
+                                !user.isAdministrator &&
+                                !user.isResearcher && (
+                                <NavbarStyledLink to='/studies'>
```

Comment:
All Studies should not be highlighted when in the achievements page. All links should be handled uniformly. A border bottom style will make all of this much simpler.

Reply:
It doesn't get highlighted, when in the achievements page

Reply:
I see, then the isActive class can be removed.